proto/store/setting.proto

```diff
@@ -60,9 +62,9 @@ message SMTPMailDeliverySetting {
   Encryption encryption = 3;
 
   // The CA, KEY, and CERT for the SMTP server.
-  string ca = 4;
-  string key = 5;
-  string cert = 6;
+  optional string ca = 4;
```

Comment:
The `encryption` field specifies whether these fields are valid or not.

Reply:
And we should validate it in the `setting_service.go`. Any issue with this field?

Reply:
I think we do not need to make it optional.

Reply:
Does it mean "" = not set?